cmd/node-termination-handler.go

```diff
@@ -373,15 +373,15 @@ func cordonNode(node node.Node, nodeName string, drainEvent *monitor.Interruptio
 		}
 		return err
 	} else {
-		log.Info().Str("node_name", nodeName).Msg("Node successfully cordoned")
+		log.Info().Str("node_name", nodeName).Str("reason", drainEvent.Kind).Msg("Node successfully cordoned")
```

Comment:
rather than using `drainEvent.Kind` can we use `drainEvent.Description`? On one hand, `Kind` seems like a minimal amount of information which is pretty nice. But the problem is that `SQS_TERMINATION` doesn't really have any meaning. When NTH is running in `sqs-processor` mode, all the reasons will be `SQS_TERMINATION` even when the SQS_TERMINATION event could actually be a SPOT_ITN, SPOT_RBR, ASG_LIFECYCLE_HOOK, etc.  

wdut?

Reply:
I agree, the SQS case specifically needs more detail. `Description` looks like a much better choice. Will update the PR to use that instead.